src/components/comments/CommentBox.jsx

```diff
@@ -14,14 +14,24 @@ class CommentBox extends React.Component {
     this.handleCommentSubmit = this.handleCommentSubmit.bind(this);
   }
 
-  getComments = () => {
+  componentDidMount() {
+    this.getComments();
+  }
+
+  handleCommentSubmit(newComment) {
+    let comments = this.state.comments;
+    let newComments = comments.concat([newComment]);
+    this.setState({ comments: newComments });
+  }
+
+  getComments() {
     const projectID = this.props.projectID;
     axios
       .get(`http://localhost:8080/api/v1/projects/${projectID}/feedback`)
```

Comment:
Can you make localhost a const like in the Navigation.jsx file, so we have that consistency throughout the repository.